benchmarks/src/stacktrace_current.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <benchmark/benchmark.h>
+#include <stacktrace>
+
+namespace {
+
+    std::stacktrace get_current(int extra_depth) {

```

Comment:
Tail call optimization may engage. Pragma optimize off, maybe?

Reply:
does this mean i need to add `#pragma optimize("", off) ... #pragma optimize("", on)` for it?

Reply:
I also notice that by adding the depth assertion I'm getting sporadic failures in the benchmarking code

(running BM_bef(150))
![image](https://github.com/microsoft/STL/assets/60953653/12ed95a0-853d-4476-a478-c6c9769bd6e2)

Reply:
Yes, that's it